### ISSUE TITLE
Song Editing

### ISSUE THREAD

**Issue by talipturkmen** [ASLP-lab/DiffRhythm#73]
Hi, Thank you for this brilliant work. I want to use it as an editing model. Does current model support inpainting and outpainting. I see that in your training code you are randomly masking audio and give as prompt like F5TTS however when I edit lyrics I hear glitches at the beginning and the end of the chunk. Is it really trained with mask or is the code just a placeholder for future work?

**Comment by NZqian** [comment]
Yeah it's an placeholder for now, see https://github.com/ASLP-lab/DiffRhythm/issues/18

**Comment by talipturkmen** [comment]
Thank you for the response. When you train with editing option did you mask the conditioning latents where you compute the loss? And do you have any planning for this feature?

**Comment by NZqian** [comment]
yes we have a mask for the conditioning latent. We are planning to release a new version of DiffRhythm in a week or two, which supports inpainting/outpainting(editing/continuation)

**Comment by jrked** [comment]
> yes we have a mask for the conditioning latent. We are planning to release a new version of DiffRhythm in a week or two, which supports inpainting/outpainting(editing/continuation)

will it have better musicality?

**Comment by talipturkmen** [comment]
Hello, is there any update on this?
